### PR TITLE
feat : Added wishlist page

### PR DESCRIPTION
**Pull request by aqsaaqeel**

### CODE REVIEW

**Review by tewarimohit — APPROVED** [review]
Code looks good to merge...

**Review by amarphule — CHANGES_REQUESTED** [review]
aq
please check all pages width is outside of view port

**Comment by tewarimohit** [comment]
Didn't see that.. nice catch amar..

File: index.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+.card-subheading{
+    font-size: 1.5rem;
+    margin: 1rem 0 0.5rem 0.5rem;
+}
+.card-body{
+    margin: 0.5rem;
+}
+.card-footer.horizontal{
+    margin-top: 2rem;
+    display: flex;
```
Comment (SanjayJatti) [inline index.css:76]: You can create utility classes for different flex as it is reused may times.

**Comment by aqsaaqeel** [comment]
> 



> aq please check all pages width is outside of view port

I'll work on the responsiveness

File: index.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    margin-top: 1rem;
+    font-size: 1rem;
+}
+.card-heading{
+    font-size: 2rem;
+    margin: 5rem 0 0 0.5rem;
+}
+.card-subheading{
+    font-size: 1.5rem;
+    margin: 1rem 0 0.5rem 0.5rem;
```
Comment (SanjayJatti) [inline index.css:69]: Create utility classes for spacing and use it as it would be more consistent.

File: index.css (hunk 1 above)
Comment (aqsaaqeel) [reply index.css:76]: Okay Ill just add it to my reset.css Thanks for the suggestion

**Pull request merged**